### PR TITLE
Added keyboard shortcuts and tooltips to buttons

### PR DESCRIPTION
**Pull request by ihabunek**

### CODE REVIEW

File: pugdebug/gui/main_window.py [hunk 1]
```diff
@@ -10,12 +10,14 @@
...
 
 from PyQt5.QtWidgets import QMainWindow, QGridLayout, QWidget, QToolBar
-from PyQt5.QtGui import QFont
+from PyQt5.QtGui import QFont, QKeySequence
 
 from pugdebug.gui.file_browser import PugdebugFileBrowser
 from pugdebug.gui.settings import PugdebugSettingsWindow
 from pugdebug.gui.workarea import PugdebugWorkareaWindow
 
+from textwrap import fill
```
Comment (robertbasic) [inline pugdebug/gui/main_window.py]: Never seen this before, what does it do?

Comment (ihabunek) [inline pugdebug/gui/main_window.py]: Wraps the text to 70 chars (default value). So the tooltips are not in one line.

Comment (robertbasic) [inline pugdebug/gui/main_window.py]: Why not make the tooltip short, and let the status tip handle the long description? http://doc.qt.io/qt-5/qaction.html#statusTip-prop

Comment (ihabunek) [inline pugdebug/gui/main_window.py]: Sounds reasonable. Will do that.

Comment (robertbasic) [inline pugdebug/gui/main_window.py]: Thanks! :)

**Comment by ihabunek** [comment]
Check it out now, updated the PR.

**Comment by robertbasic** [comment]
Any reason for not adding it to start and stop actions? :)

**Comment by ihabunek** [comment]
Ehh, actually wanted shortcuts and then as an afterthought added tips, so
forgot about start and stop.

Leave the PR open and I'll do it tomorrow.

**Comment by robertbasic** [comment]
OK! Sorry for nitpicking so much!

**Comment by ihabunek** [comment]
No problem, nitpicking is what get things done. :)
Added stuff for start and stop.

**Comment by robertbasic** [comment]
Thanks! :)

**Pull request merged**